cli/cli.go

```diff
@@ -30,7 +31,8 @@ func Execute(appName, version string) {
 }
 
 func init() {
-	RootCmd.AddCommand(bootnode.StartBootNodeCmd)
 	RootCmd.AddCommand(operator.StartNodeCmd)
 	RootCmd.AddCommand(operator.GenerateDocCmd)
+	RootCmd.AddCommand(bootnode.StartBootNodeCmd)
+	RootCmd.AddCommand(tests.RsaCmd)
```

Comment:
should be a completely separate command if possible IMO, no need to add it to the RootCmd, make its own.